qcelemental/models/molecule.py

```diff
@@ -186,25 +194,27 @@ class Molecule(ProtoModel):
         )
     fragments: Optional[List[Array[np.int32]]] = Schema(
```

Comment:
I thought @dgasmith ended up with fragment* fields remaining lists, not converting to ndarray?

Reply:
This signature is correct, I gave up and went with it.